.github/workflows/CI.yml

```diff
@@ -116,3 +116,54 @@ jobs:
       - name: Test
         run: |
           ${{ matrix.config.qemu }} build/uv_run_tests_a
+
+  build-cross-mingw:
+    runs-on: ubuntu-latest
+    name: build-cross-mingw
+    steps:
+      - uses: actions/checkout@v2
+      - name: Envinfo
+        run: npx envinfo
+      - name: Install mingw32 environment
+        run: |
+          sudo apt update
+          sudo apt install mingw-w64 -y
+      - name: Build
+        run: |
+          mkdir build
+          cd build && cmake .. -DBUILD_TESTING=ON -DQEMU=ON -DCMAKE_TOOLCHAIN_FILE=../cmake-toolchains/cross-mingw32-linux.cmake
+          cmake --build .
```

Comment:
Would adding ```cmake --install . --prefix "`pwd`/usr"``` here be useful, and then only upload the installation files from `build/usr/**/*`?

Reply:
Maybe, yes. I can try this too.